NEWS.md

```diff
@@ -1,5 +1,10 @@
 # rsconnect (development version)
 
+* `deployApp()` and `writeManifest()` now error if your library and lockfile
```

Comment:
`renv.lock` rather than "lockfile".

"should err" or "should error"? I prefer "err", but others see that as an "error". 😜

Reply:
I feel like "err" means to make a mistake; whereas here we are erroring (or more technically correct, throwing an error).